javascript/GridField.js

```diff
@@ -136,6 +136,11 @@
 		$('.ss-gridfield .action').entwine({
 			onclick: function(e){
 				var filterState='show'; //filterstate should equal current state.
+
+				// If the button is disabled, do nothing.
+				if (this.button('option', 'disabled')) {
+					return;
```

Comment:
Replacing the `return` with `e.preventDefault()` won't work. At the bottom of the handler `this.getGridField().reload()` will still get called unless you return. I could use both but I'm not sure what default action would be cancelled here?